recipe/meta.yaml

```diff
@@ -10,30 +10,44 @@ source:
     url: https://github.com/IntelPython/{{ name }}/archive/{{ version }}.tar.gz
     sha256: {{ sha256 }}
     folder: "{{ name|lower }}-{{ version }}"
+    patches:
+        - 0001-fix-build-breakage-with-update-of-skbuild.patch
+        - 0002-remove-pybind-fetch.patch
 build:
     number: 0
-    skip: True  # [not (x86_64 and (linux or win))]
+
+    # Only supported on intel compilers
+    # Some syntax used in dpctl is only supported from python 3.8
+    # A rebuild against mkl 2022 must happen before we can build dpctl for windows.
+    skip: True  # [(not (x86_64 and (linux or win))) or py<38 or win]
+
     script_env:
         - ONEAPI_ROOT
+    missing_dso_whitelist: # [linux]
+        - $RPATH/lib*      # [linux]
     ignore_run_exports:
         - dpcpp-cpp-rt  # follow intel channel's package and set this in run.
 
 requirements:
     build:
-        - {{ compiler('cxx') }}
-        - {{ compiler('dpcpp') }}  # [not osx]
+        - {{ compiler('c') }}
+        - {{ compiler('dpcpp') }}
+        - patch     # [not win]
+        - m2-patch  # [win]
     host:
         - setuptools
         - cython
-        - cmake
+        - cmake  >=3.21
+        - pybind11 2.9.2
+        - git
         - python
-        - make                     # [unix]
-        - ninja                    # [win]
-        - numpy 1.19
+        - ninja
+        - scikit-build
+        - numpy 1.21
         - wheel
     run:
         - python
-        - {{ pin_compatible('numpy') }}
+        - {{ pin_compatible('numpy', min_pin='x.x', max_pin='x') }}
```

Comment:
use here please instead 'numpy >=1.21,<2'.  We want to get rid of 'pin_compatible'